### PR TITLE
Add warning for advanced network configurations

### PR DESCRIPTION
**Pull request by J-N-K**
Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.sonos/README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 This binding integrates the [Sonos Multi-Room Audio system](http://www.sonos.com).
 
+**Attention:** 
+You might run into trouble if your Eclipse Smarthome instance is in another subnet than your Sonos device.
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.sonos/README.md]: Better talk of "your gateway" or something neutral - remember that those docs are e.g. taken over to the openHAB documentation website.

Comment (J-N-K) [reply extensions/binding/org.eclipse.smarthome.binding.sonos/README.md]: I was thinking about that, too. But „gateway“ is not a good idea either as that is widely used in another context in networking.

Comment (kaikreuzer) [reply extensions/binding/org.eclipse.smarthome.binding.sonos/README.md]: Then simply use "instance".

**Comment by sjka** [comment]
After stuggling a bit stetting up my system using an IGMP proxy in between two VLANs I gave up. I couldn't get it to work, although the idea is pretty clear though... Do you maybe have a pointer to how you got it working?

**Pull request merged**